packages/riipen-ui/src/components/InputLabel.test.jsx

```diff
@@ -0,0 +1,224 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import InputLabel from "./InputLabel.jsx";
+
+describe("<InputLabel>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<InputLabel />);
+    } catch (e) {
+      error = e;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<InputLabel />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const defaultProps = new InputLabel().type.defaultProps;
+
+      const wrapper = mount(<InputLabel />);
+
+      const component = wrapper.find("InputLabel");
+      expect(component.props().color).toEqual(defaultProps.color);
+      expect(component.props().required).toEqual(defaultProps.required);
+      expect(component.props().weight).toEqual(defaultProps.weight);
+    });
+  });
+
+  describe("children prop", () => {
+    it("renders given children inside the label and Typography element", () => {
+      const child = <div>it's a label</div>;
+
+      const wrapper = mount(<InputLabel children={child} />);
+
+      expect(
+        wrapper
+          .find("InputLabel")
+          .find("label")
+          .find("Typography")
+          .contains(child)
+      ).toEqual(true);
+    });
+
+    it("does not render label and Typography elements when children is not given", () => {
+      const wrapper = mount(<InputLabel />);
+
+      expect(wrapper.find("InputLabel").find("label")).toHaveLength(0);
+      expect(wrapper.find("InputLabel").find("Typography")).toHaveLength(0);
```

Comment:
Same here. We can just remove this. 

I think this is slightly better because testing for the typography is just a subset of testing for the label, so we are essentially double testing here.